# Homework 4

**Name:** Ji Jie


## Instructions for typesetting math in Markdown cells

It is possible to render mathematical expressions nicely in Markdown cells, using a type of script called "LaTex". LaTex the standard format for people in most technical fields to write math nicely. (You can Google for examples of what you can do with LaTex.) 

To render an expression using LaTex, the expression must be enclosed with dollar signs. For example, the expression `$X>0$` is rendered $X>0$. A single dollar sign is for mathematical expressions within the same line, and double dollar signs are for a standalone mathematical expressions in its own line. Hence, `$$X > 0.$$` is rendered as  $$X>0.$$

Below are examples of how to typeset various mathematical symbols using Latex (the math itself is not that meaningful, but I wanted to showcase all the commands you would need). For additional examples, see the course handouts (the .ipynb files from nbviewer), or Google for "latex math examples."

Let $A = 3/4 = \frac{3}{4} =  0.75$, $B=\mathbb{P}(A \ge 1) = 0$. The expectation $\mathbb{E}(AB)=0$. It's okay also to write $E(AB)$ instead of $\mathbb{E}(AB)$, or $P(A \ge 1)$ instead of $\mathbb{P}(A \ge 1)$ although the later look nicer. You can write a sequence of equations using the align environment, which uses `\\` for breaking a line and `&` for alignment. Note that when you are in math mode (within dollar signs), you must use `\text{}` to enclose text.
$$\begin{aligned}
P(A \ge 80\% \text{ and } Y \approx \sqrt{3} | X \text{ is red}) & = \frac{ P(A \ge 80\% \text{ and } Y \approx \sqrt{3} \text{ and $X$ is red})}{P(X \text{ is red})} \\
&\le \frac{ P(A \ge .8)}{P(X \text{ is red})} \\
&\le 0.
\end{aligned}$$


Let $A = 3/4 = \frac{3}{4} =  0.75$, $B=\mathbb{P}(A \ge 1) = 0$. The expectation $\mathbb{E}(AB)=0$. It's okay also to write $E(AB)$ instead of $\mathbb{E}(AB)$, or $P(A \ge 1)$ instead of $\mathbb{P}(A \ge 1)$ although the later look nicer. You can write a sequence of equations using the align environment, which uses `\\` for breaking a line and `&` for alignment. Note that when you are in math mode (within dollar signs), you must use `\text{}` to enclose text.
$$\begin{aligned}
P(A \ge 80\% \text{ and } Y \approx \sqrt{3} | X \text{ is red}) & = \frac{ P(A \ge 80\% \text{ and } Y \approx \sqrt{3} \text{ and $X$ is red})}{P(X \text{ is red})} \\
&\le \frac{ P(A \ge .8)}{P(X \text{ is red})} \\
&\le 0.
\end{aligned}$$


## Q1. DMD Exercise 2.1 

A four-sided die is engraved with the numbers 1 through 4 on its four different sides. Suppose that when rolled, each side (and hence each number) has an equal probability of being the bottom face when it lands. We roll two such dice. Let X be the sum of the numbers on the bottom faces of the two dice.

(a) What is the probability that X is at least five?
$$\begin{aligned}
P(X \ge 5) & = \frac{5}{8}
\end{aligned}$$
(b) How does your answer to (a) change if you are told that the bottom face of the first die has the number 3 on it?
$$\begin{aligned}
P(X \ge 5 | X_{1} = 3) & = \frac{3}{4}
\end{aligned}$$
(c) How does your answer to (a) change if you are told that the bottom face of one of the dice has the number 3 on it?
$$\begin{aligned}
P(X \ge 5 | X_{1} \text{ or } X_{2} = 3) & = \frac{5}{7}
\end{aligned}$$

## Q2. DMD Exercise 2.4 

An oil company is drilling for oil at three promising sites. According
to geological tests, the probabilities of finding oil at these three sites are 0.70, 0.85, and 0.80,
respectively. The presence of oil at any one of the sites is presumed to be independent of the
presence of oil at any of the other sites.

(a) What is the probability of finding oil at all three of the sites?
$$\begin{aligned}
P(X) & = 0.476
\end{aligned}$$
(b) What is the probability of not finding oil at any of the three sites?
$$\begin{aligned}
P(X) & = 0.009
\end{aligned}$$


In [4]:
0.7*0.85*0.8

0.476

In [5]:
0.3*0.15*0.2

0.009

## Q3. DMD Exercise 2.10 
A hardware store has received two shipments of halogen lamps. The first shipment contains 100 lamps, 4 percent of which are defective. The second shipment contains 50 lamps, 6 percent of which are defective. Suppose that Emanuel picks a lamp (at random) off of the shelf and purchases it, and he later discovers that the lamp he purchased is defective. Is the defective lamp more likely to come from the first shipment or from the second shipment?

A: More likely come from first shipment


In [6]:
(0.08/3) / (7/150)

0.5714285714285714

In [7]:
(0.06/3) / (7/150)

0.42857142857142855

In [8]:
2/3 * 0.1

0.06666666666666667

In [9]:
9/29

0.3103448275862069

## Q4. DMD Exercise 2.11 
It is estimated that one third of the population in a given county is infected with the tuberculosis (TB) bacteria. The human body is usually able to successfully fight the TB bacteria and so prevent the onset of the TB disease. Consequently, a person infected with the TB bacteria has only a 10 percent chance of development the TB disease over his/her lifetime.

(a)	Suppose that we choose a person at random from all of the people in the county. What is the probability that this person will have TB disease over his/her lifetime?
$$\begin{aligned}
P(X) & = 0.0333
\end{aligned}$$
(b)	Suppose that among those people who have died in the county in the last year, we perform an autopsy on one of the bodies chosen at random, and we find that this person did not have the TB disease. What is the probability that this person had been infected with the TB bacteria?
$$\begin{aligned}
P(X) & = 0.3103
\end{aligned}$$

## Q5. (Geographic Testing)

Extensive logging has exposed a hillside in San Carlos to the possibility of a mudslide. The mayor has consulted a geographic expert, who estimates that there is only a 10\% chance that the slide will occur within the next year. The expert adds that roughly 5\% of such slides would break through the existing retaining wall, which would cause a damage of one million dollars. (The slides that don't break the wall would cause negligible damage.) 

The expert also mentioned that she can conduct a geological test to better assess the likelihood of a mud slide. The test has three possible outcomes, as seen in the table below, which also summarize past data (from other locations) on whether a subsequent slide indeed happened afterward.

| Test Outcome | Slide within 1 year | No slide within 1 year | 
|--|--|--|
|Positive| 900 | 50 | 
|Neutral| 25 |  100 |
|Negative| 75 | 850 |

**(a)** Suppose that the expert's initial estimate of 10\% chance is correct, what's the probability that the test would yield a positive result in San Carlos?
$$\begin{aligned}
P(X) & = 0.135
\end{aligned}$$

**(b)** If there is a way to strengthen the retaining wall so that it would not break under any mud slide, how much should the mayor be willing to pay for the wall supposing that the geographic test yields a positive result? What about if it yields a neutral result?
$$\begin{aligned}
C(Positive) = 33333.33
\end{aligned}$$
$$\begin{aligned}
C(Neutral) = 1351.35
\end{aligned}$$

a) given p(slide) = 0.1
p(pos|slide)=0.9
p(pos|not slide)=0.05

need
p(positive)=? = p(pos&slide) + p(pos&no slide) = 0.1*0.9+0.9*0.05

b) E[benifit] VS cost\

E[benifit] = 0.05*1000000*p(slide|pos)

C) value of information is what you can gain knowing it\
No test  (Not build)\
Test (pos, neu, nega)

In [24]:
2/3 * 0.05 * 1000000 * 0.135

4500.000000000001

In [25]:
0.0025 / 0.0925 *0.05*1000000 * 0.0925

125.00000000000001

In [19]:
0.85*0.9 + 0.0075

0.7725

In [18]:
75/1000 * 0.1

0.0075

In [29]:
0.0075/0.7725 * 0.05 * 1000000 * 0.7725

375.00000000000006

In [28]:
0.1*0.05*1000000

5000.000000000001

In [26]:
2/3 * 0.135 + 0.0025/0.0925 * 0.0925 + 0.0075/0.7725 * 0.7725

0.1

**(c)** Suppose the way to strengthen the retaining wall costs $20000$ dollars, how much should the mayor be willing to pay for the geological test?

In [13]:
(25/1000) * 0.1 / 0.2

0.012500000000000002

In [14]:
(75/1000) * 0.1 / (75/925)

0.09249999999999999